p2p/simulations/network.go

```diff
@@ -388,7 +399,7 @@ func (self *Network) Stop(id discover.NodeID) error {
 	node.Up = false
 	log.Info(fmt.Sprintf("stop node %v: %v", id, node.Up))
 
-	self.events.Send(ControlEvent(node))
+	self.events.Send(NewEvent(node))
```

Comment:
I propose not making this change (it is a control event, not a live event), and fixing the actual live event.